### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by parasew**
updated the readme to point to the latest version of QT 5.8 on OSX (on current qt.io installer)

### CODE REVIEW

**Comment by parasew** [comment]
on another note: the correct path description for OSX would be 
`PATH=$HOME/Qt/5.8/clang_64/bin:$PATH`
otherwise the build script still takes the system or homebrew version of qt!

**Comment by fluffypony** [comment]
@parasew please rebase

File: README.md [hunk 1]
```diff
@@ -118,9 +118,9 @@ Packaging for your favorite distribution would be a welcome contribution!
     `brew install cmake`
 
 4. Install latest Qt using official installer from [qt.io](https://www.qt.io/download-open-source/) (homebrew version might be outdated).
-5. Add Qt bin dir to your path:
+5. Add Qt bin dir to your path (check first if the directory is existing; 5.8 is the current version)
```
Comment (NanoAkron) [inline README.md]: *'first check if the directory exists' is better English

**Comment by fluffypony** [comment]
@parasew ping - please rebase:)

**Comment by parasew** [comment]
Hi Ricardo! Rebase is done. All the best, Matthias

**Pull request merged**